### PR TITLE
re-org moconi/marconi-mamba test folders [PLT-1360]

### PR DESCRIPTION
**Pull request by kayvank**
Re org the test test modules to match the corresponding module name under test
Re org marconi-mamba utxo related module names to match those of marconi

[PLT-1360]

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Formatting, PNG optimization, etc. are updated
    - [ ] Important changes are reflected in changelog.d of the affected packages
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
    - [ ] Reference the ADR in the PR and reference the PR in the ADR (if revelant)
    - [ ] Reviewer requested

[PLT-1360]: https://input-output.atlassian.net/browse/PLT-1360?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[PLT-1317]: https://input-output.atlassian.net/browse/PLT-1317?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by raduom — APPROVED** [review]

**Review by koslambrou — APPROVED** [review]

File: marconi-mamba/marconi-mamba.cabal [hunk 1]
```diff
@@ -187,22 +183,26 @@ test-suite marconi-mamba-test
   type:           exitcode-stdio-1.0
   main-is:        Spec.hs
   hs-source-dirs: test
-  other-modules:  Spec.UtxoIndexersQuery
+  other-modules:  Test.Marconi.Api.Index.Utxo
+
+  --------------------------
+  -- Other IOG dependencies
+  --------------------------
+  build-depends:  cardano-api:{cardano-api, gen}
```
[the remark this answers is not in the document]
Comment (kayvank) [reply marconi-mamba/marconi-mamba.cabal:191]: cardano-api should be under local? 
We have it under `Other IOG dependencies `in other projects

[the remark this answers is not in the document]
Comment (koslambrou) [reply marconi-mamba/marconi-mamba.cabal:195]: Oops, my bad :D

File: marconi-mamba/marconi-mamba.cabal [hunk 2]
```diff
@@ -134,7 +134,7 @@ executable examples-json-rpc-client
   --------------------
   build-depends:  marconi:{marconi, json-rpc}
 
-  ------------------------
+  --------------
```
Comment (koslambrou) [inline marconi-mamba/marconi-mamba.cabal:137]: Remove the diff of this line

File: marconi-mamba/marconi-mamba.cabal (hunk 1 above)
[the remark this answers is not in the document]
Comment (koslambrou) [reply marconi-mamba/marconi-mamba.cabal:195]: Ahh, what I meant was: Move the `Other IOG dependencies` under the `Local components` section.

File: marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/ScriptTx.hs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import Test.Tasty (TestTree, testGroup)
+import Test.Tasty.Hedgehog (testPropertyNamed)
+
+import Cardano.Api qualified as C
+import Gen.Cardano.Api.Typed qualified as CGen
+import Gen.Marconi.ChainIndex.Types (genTxBodyWithTxIns, genWitnessAndHashInEra)
+import Marconi.ChainIndex.Indexers.ScriptTx qualified as ScriptTx
+
+tests :: TestTree
+tests = testGroup "Marconi.ScriptTx.Indxers.Specs are:"
```
Comment (koslambrou) [inline marconi-chain-index/test/Spec/Marconi/ChainIndex/Indexers/ScriptTx.hs:22]: Change to `Spec.Marconi.ChainIndex.Indexers.ScriptTx`

File: marconi-mamba/marconi-mamba.cabal (hunk 1 above)
[the remark this answers is not in the document]
Comment (koslambrou) [reply marconi-mamba/marconi-mamba.cabal:194]: @kayvank That wasn't resolved.

File: marconi-mamba/test/Spec/Marconi/Mamba/Api/Query/Indexers/Utxo.hs [hunk 4]
```diff
@@ -5,34 +5,89 @@
...
+  let rows = concatMap Utxo.eventsToRows events
+  fetchedRows === rows
+
+addressAnyToShelley :: C.AddressAny -> Maybe (C.Address C.ShelleyAddr)
+addressAnyToShelley  (C.AddressShelley a ) = Just a
+addressAnyToShelley  _                     = Nothing
+
+
+
+{-
```
Comment (koslambrou) [inline marconi-mamba/test/Spec/Marconi/Mamba/Api/Query/Indexers/Utxo.hs:90]: bin?

**Review by koslambrou — APPROVED** [review]
LGTM. Minor fixes.

**Pull request merged**